docs/data-sources/gnomad-small-variants-json.md

```diff
@@ -0,0 +1,102 @@
+   "amrAn":832,
+   "easAn":1618,
+   "finAn":3486,
+   "nfeAn":14916,
+   "asjAn":302,
+   "othAn":978,
+   "allAc":5861,
+   "maleAc":2930,
+   "femaleAc": 2931,
+   "afrAc":1931,
+   "amrAc":101,
+   "easAc":388,
+   "finAc":477,
+   "nfeAc":2704,
+   "asjAc":78,
+   "othAc":182,
+   "allHc":561,
+   "afrHc":208,
+   "amrHc":6,
+   "easHc":42,
+   "finHc":31,
+   "nfeHc":242,
+   "asjHc":13,
+   "othHc":19,
+   "maleHc":280,
+   "femaleHc":281,
+   "controlsAllAf":0.190317,
+   "controlsAllAn":30796,
+   "controlsAllAc":5861,
+   "lowComplexityRegion":true,
+   "failedFilter":true
+}
+```
+| Field               | Type   | Notes                                                                                            |
+|---------------------|--------|--------------------------------------------------------------------------------------------------|
+| coverage            | int    | average coverage (non-negative integer values)                                                   |
+| allAf               | float  | allele frequency for all populations. Range: 0 - 1.0                                             |
+| maleAf              | float  | allele frequency for male population. Range: 0 - 1.0                                             |
+| femaleAf            | float  | allele frequency for female population. Range: 0 - 1.0                                           |
+| controlsAllAf       | float  | allele frequency for the controls subset. Range: 0 - 1.0                                         |
+| allAc               | int    | allele count for all populations. Integer.                                                       |
+| maleAc              | int    | allele count for male population. Integer.                                                       |
+| femaleAc            | int    | allele count for female population. Integer.                                                     |
+| controlsAllAc       | int    | allele count for the controls subset. Integer.                                                   |
+| allAn               | int    | allele number for all populations. Non-zero integer.                                             |
+| maleAn              | int    | allele number for male population. Non-zero integer.                                             |
+| femaleAn            | int    | allele number for female population. Non-zero integer.                                           |
+| controlsAllAn       | int    | allele number for the controls subset. Non-zero integer.                                         |
+| allHc               | int    | count of homozygous individuals for all populations. Non-negative integer.                       |
+| maleHc              | int    | count of homozygous individuals for male population. Non-negative integer.                       |
+| femaleHc            | int    | count of homozygous individuals for female population. Non-negative integer.                     |
```

Comment:
Capitalize first letter?

Reply:
@rajatshuvro Actually, when I looked at the "Notes" from the other tables, most of them start with lower case letters.